hail/src/main/scala/is/hail/asm4s/FunctionBuilder.scala

```diff
@@ -328,7 +343,7 @@ class FunctionBuilder[F >: Null](val parameterTypeInfo: Array[MaybeGenericTypeIn
       cn.accept(cw)
       bytes = cw.toByteArray
 //       This next line should always be commented out!
```

Comment:
Can we get rid of this comment?

Reply:
no, the line below should be commented out! This is fixed in the parent PR (rvb copy)